releases/release-6.5.3.md

```diff
@@ -0,0 +1,168 @@
+---
+title: TiDB 6.5.3 Release Notes
+summary: 了解 TiDB 6.5.3 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 6.5.3 Release Notes
+
+发版日期：2023 年 x 月 x 日
+
+TiDB 版本：6.5.3
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.5.3#version-list)
+
+## 兼容性变更
+
+- note 1
+
+## 改进提升
+
++ TiDB
+
+    <!--tw:ran-huang-->
+    - 提升了 TRUNCATE 分区表（带放置规则）操作的速度。[#43070](https://github.com/pingcap/tidb/issues/43070) @[Lloyd-Pottiger](https://github.com/Lloyd-Pottiger)
+    - 在 resolve lock 之后避免 stale-read 的无效重试。[#43659](https://github.com/pingcap/tidb/issues/43659) @[you06](https://github.com/you06)
+    - 为 partition table 设置更多的并发度来加速 limit 语句下的执行。[#41480](https://github.com/pingcap/tidb/issues/41480) @[you06](https://github.com/you06)
+
```

Comment:
```suggestion
    - 在 stale-read 遇到 data-is-not-ready 时使用 leader read 降低延迟。#[765](https://github.com/tikv/client-go/pull/765) @[Tema](https://github.com/Tema)

```

This is patched to TiDB by upgrading client-go in a PR, so there is no direct issue of it. @ran-huang should it be here?

Reply:
That's ok.